src/main/resources/static/api.yml

```diff
@@ -1635,6 +1635,9 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Characteristic'
+        status:
```

Comment:
Think this ought to be a `'#/components/schemas/PropertyStatus'` too so it's clear from Swagger which values are acceptable